### PR TITLE
E04 04 blog display

### PR DESCRIPTION
**Pull request by AnasMama**
# TASK TO EPIC PR

## The feature / The problem
Implementation of 2 blog templates and the editing menu

## Scope of my changes
Route /blogs

## Task
- [e04-t04:](https://www.notion.so/e04-04_blog-display-2ae1efceafa44101a25d63505bdaa859)

## Type of change
- [x] New feature (feat)
- [ ] Fix issue (fix)
- [ ] Code refactorisation (refactor)
- [ ] Code testing (test)
- [ ] Add documentation (docs)

# Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my PR
- [x] I have removed not needed logs
- [x] I have remover TODO comments
- [ ] I have commented in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have been able to build my solution locally
- [x] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes

# Communication
Something special like a warning about this PR

### CODE REVIEW

File: src/routes/blogs/BlogT1.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+}) => {
+  return (
+    <>
+      <main className="relative min-h-screen w-full max-w-screen-2xl mx-auto my-8 flex flex-col items-center gap-8">
+        <header className="h-96 w-full m-auto bg-[url('https://placeimg.com/1000/800/arch')] bg-opacity-25 flex flex-col justify-center items-center text-white gap-4">
+          <h1 className="text-7xl font-bold font-lobster bg-neutral/80 p-2">
+            Voyage machin
+          </h1>
+          <p className="max-w-2xl bg-neutral/80 p-2">
+            Si vous êtes passionné de voyage et que vous avez envie de partager
```
Comment (Holmes-EH) [inline src/routes/blogs/BlogT1.tsx:18]: Il est bon ce chatGPT ! 😜

File: src/routes/blogs/BlogT2.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
+import Pagination from '../../components/buttons/Pagination'
+import CardT2 from '../../components/CardT2'
+import SearchBar from '../../components/inputs/SearchBar'
+
+const BlogT1 = ({
```
Comment (Holmes-EH) [inline src/routes/blogs/BlogT2.tsx:5]: Copy pasta ?
```suggestion
const BlogT2 = ({
```

File: src/routes/blogs/BlogT2.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            </div>
+            <Pagination />
+          </article>
+        </section>
+      </main>
+    </>
+  )
+}
+
+export default BlogT1
```
Comment (Holmes-EH) [inline src/routes/blogs/BlogT2.tsx:76]:
```suggestion
export default BlogT2
```

File: src/routes/createblog.tsx [hunk 4]
```diff
@@ -77,6 +78,7 @@ const CreateBlog = () => {
 
   const createNewBlog = () => {
     setIsCreatingBlog(false)
+    console.log(user)
```
Comment (Holmes-EH) [inline src/routes/createblog.tsx:81]:
```suggestion

```

File: src/App.tsx [hunk 5]
```diff
@@ -22,8 +23,9 @@ function App() {
       {message && <Toaster />}
       <Routes>
         <Route element={<Home />} path="/" />
+        <Route element={<BlogName />} path="/test" />
```
Comment (Holmes-EH) [inline src/App.tsx:26]: J'ai typescript qui se plaint sur cette ligne :
```
Property 'setIsEditing' is missing in type '{}' but required in type '{ setIsEditing: Dispatch<SetStateAction<boolean>>; }'.ts(2741)
BlogT1.tsx(8, 3): 'setIsEditing' is declared here.
```
Mais je suppose que c'est du temp au vu du `path` ?

**Review by Holmes-EH — APPROVED** [review]
@AnasMama , tu nous a fait une insomnie ! Tu voulais voir le Père Noël, c'est ça ? 😆 
Il y a quelques commentaires sur ma review, mais j'approuve, puisque c'est Noël !.. 🎄 🎁 
Mais beau boulot quand même !
Maintenant, profites, et passe un bon réveillon de la Saint-Sylvestre !! 🎆

**Review by Antoniatch — APPROVED** [review]

**Review by Tbiarneix — APPROVED** [review]

**Pull request merged**